src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -344,6 +346,11 @@ public enum CassandraRelevantProperties
      */
     GOSSIP_SETTLE_POLL_SUCCESSES_REQUIRED("cassandra.gossip_settle_poll_success_required", "3"),
 
+    CONSISTENT_RANGE_MOVEMENT("cassandra.consistent.rangemovement", "true"),
+    CONSISTENT_SIMULTANEOUS_MOVES_ALLOW("cassandra.consistent.simultaneousmoves.allow"),
+    REPLACE_ADDRESS("cassandra.replace_address"),
+    BROADCAST_INTERVAL_MS("cassandra.broadcast_interval_ms", "60000"),
```

Comment:
Yes, all of them are present in 5.0. I just followed what we have today in 4.1 - a bit of mess, variables are defined but not used consistently. I can revert if you like

Reply:
` I just followed what we have today in 4.1 - a bit of mess, variables are defined but not used consistently.`
The story is different. The original ticket (CASSANDRA-15876) where `CassandraRelevantProperties` was created added only those system properties already added in the VT. Which other properties will be added to the class and exposed to the VT was a matter of a follow-up ticket (CASSANDRA-17797). The batch of added properties was used around the codebase (in case someone didn't miss anything)